### PR TITLE
New web module for brighter fonts in navigation 

### PR DESCRIPTION
**Pull request by eqms**
In case of working on a very bright working station, as well as presentation with an projector, may be difficult to see the links in the main-navigation.
With Clean Navbar, you get a brighter main-navigation.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Thanks for the contribution.

It seems good in general, but please remove company logo, and put image inside README.rst as sample. This image cropped can be also used as module icon.

**Comment by dreispt** [comment]
Suggestion: it would be nice to add screenshots with before and after examples.

**Comment by legalsylvain** [comment]
@dreispt  web_clean_navbar/static/description/navbar1.png like that ?

**Comment by dreispt** [comment]
@legalsylvain Yes, you're right, sorry for missing that.

**Comment by gurneyalex** [comment]
@eqjm and @eqms CLA checked

File: web_clean_navbar/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#
+##############################################################################
+
+{
+    'name': 'Web Clean Navbar',
+    
+    'version': '1.0.0',
+    'category': 'Web',
+    'description': """
+        Better visibility for the main backend navigation-bar
```
Comment (hbrunn) [inline web_clean_navbar/__openerp__.py:28]: Please put that into README.rst

File: web_clean_navbar/__init__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
```
Comment (hbrunn) [inline web_clean_navbar/__init__.py:21]: remove blank line at end of file

File: web_clean_navbar/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Web Clean Navbar',
+
```
Comment (hbrunn) [inline web_clean_navbar/__openerp__.py:24]: blank line contains whitespace

**Comment by hbrunn** [comment]
@eqms great module, thanks! Please replace your company logo by something depicting what the module does or simply use the OCA logo from https://github.com/OCA/maintainer-tools/tree/master/template/module/static/description (we agreed in the OCA not to use contributed modules as advertising for the contributor, save for the author/maintainer/website fields of course)

**Comment by hbrunn** [comment]
closing in favor of #223

**Pull request closed without merging**